### PR TITLE
include importance on monitoring

### PR DESCRIPTION
**Pull request by RogerRussel**
It will allow put importance on monitoring system, like if something is huge important or low importance.

### CODE REVIEW

File: src/Observer.php [hunk 1]
```diff
@@ -5,26 +5,28 @@
...
+  const IMPORTANCE_ALIAS = 'importance';
 
-    date_default_timezone_set($conf['timezone']);
+  public function __construct($conf){
 
     $this->start = microtime();
 
     $this->conf = $conf;
+
+    $this->applaySettings();
```
Comment (alanyoshida) [inline src/Observer.php]: Apply ?

File: src/Observer.php [hunk 2]
```diff
@@ -69,6 +73,33 @@ public function runnit($name, $conf){
 
   }
 
+  protected fucntion applaySettings(){
```
Comment (alanyoshida) [inline src/Observer.php]: Apply?

Comment (RogerRussel) [inline src/Observer.php]: Yep, it should be apply, and not some thing like ap play settings.

**Comment by RogerRussel** [comment]
I didn't use priority because, I was worried about misunderstanding it to be a run priority, but now I think it doesn't have difference at all.
Which is better? What is it priority on keep running or what is it importance on keep running?

File: src/Observer.php [hunk 3]
```diff
@@ -69,6 +73,33 @@ public function runnit($name, $conf){
 
   }
 
+  protected fucntion applySettings(){
```
Comment (alanyoshida) [inline src/Observer.php]: fucntion should be function

**Comment by RogerRussel** [comment]
It's alright to merge?

**Comment by alanyoshida** [comment]
Aproved !

**Pull request merged**